lib/std/zig/render.zig

```diff
@@ -1295,8 +1295,16 @@ fn renderExpression(
                 try renderExpression(allocator, stream, tree, indent, start_col, lib_name, Space.Space);
             }
 
+            // Some extra machinery is needed to rewrite the old-style cc
+            // notation to the new callconv one
```

Comment:
Thanks for adding this auto-upgrading functionality to zig fmt, it's quite nice. Would you mind adding a `testTransform` to the top of parser_test.zig along with a TODO, to remind us to remove this functionality after 1 release cycle?

Reply:
Will do!